content/en/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -53,63 +50,38 @@ You can classify requests based on their type, for example `ListReview`,
     service-specific.
 
     {{< text yaml >}}
-apiVersion: networking.istio.io/v1alpha3
-kind: EnvoyFilter
+apiVersion: extensions.istio.io/v1alpha1
+kind: WasmPlugin
 metadata:
   name: istio-attributegen-filter
 spec:
-  workloadSelector:
-    labels:
+  selector:
+    matchLabels:
       app: reviews
-  configPatches:
-  - applyTo: HTTP_FILTER
-    match:
-      context: SIDECAR_INBOUND
-      proxy:
-        proxyVersion: '1\.9.*'
-      listener:
-        filterChain:
-          filter:
-            name: "envoy.filters.network.http_connection_manager"
-            subFilter:
-              name: "istio.stats"
-    patch:
-      operation: INSERT_BEFORE
-      value:
-        name: istio.attributegen
-        typed_config:
-          "@type": type.googleapis.com/udpa.type.v1.TypedStruct
-          type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
-          value:
-            config:
-              configuration:
-                "@type": type.googleapis.com/google.protobuf.StringValue
-                value: |
-                  {
-                    "attributes": [
-                      {
-                        "output_attribute": "istio_operationId",
-                        "match": [
-                          {
-                            "value": "ListReviews",
-                            "condition": "request.url_path == '/reviews' && request.method == 'GET'"
-                          },
-                          {
-                            "value": "GetReview",
-                            "condition": "request.url_path.matches('^/reviews/[[:alnum:]]*$') && request.method == 'GET'"
-                          },
-                          {
-                            "value": "CreateReview",
-                            "condition": "request.url_path == '/reviews/' && request.method == 'POST'"
-                          }
-                        ]
-                      }
-                    ]
-                  }
-              vm_config:
-                runtime: envoy.wasm.runtime.null
-                code:
-                  local: { inline_string: "envoy.wasm.attributegen" }
+  url: https://storage.googleapis.com/istio-build/proxy/attributegen-359dcd3a19f109c50e97517fe6b1e2676e870c4d.wasm
```

Comment:
should we move source code to https://github.com/istio-ecosystem/wasm-extensions?

Reply:
Yes, I was going to do that.

Reply:
Build is broken there, so I'll put source there for now.

Reply:
https://github.com/istio-ecosystem/wasm-extensions/pull/85

Reply:
Understand this is deprecated and we are removing references. But do we also have to add a new document which explains what is the new way of doing things? Also if this whole page is deprecated or a sub-section, makes sense to add a deprecated warning banner, if not already done.

Reply:
@kfaseela We should make a big pass over docs and replace all wasm and telemetry EnvoyFilter examples with Telemetry API and Wasm API. That should be a follow up before the next release (should have really done it for 1.17).

Reply:
Agree about creating a blocking issue. The only reason I found this reference is because it would break on the next update of proxy configs.

Reply:
@ericvn :  the lint fails on [the PR ](https://github.com/istio/istio.io/pull/12772) waiting for this PR